test/unittests/execute_floating_point_test.cpp

```diff
@@ -100,13 +88,140 @@ struct WasmTypeName
     }
 };
 
+template <typename T>
+class execute_floating_point_types : public testing::Test
+{
+protected:
+    using L = typename FP<T>::Limits;
+
+    // The q is any positive floating-point value without zeros, infinities and NaNs.
+    inline static const std::array q_values{
```

Comment:
What's the point of making it inline?

Reply:
I can define it here instead of doing so outside of the class.